holidays/countries/cocos_islands.py

```
@@ -0,0 +1,212 @@
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from datetime import date, timedelta
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.utils import nth_weekday_of_month
```

Ruff (0.8.2)

`holidays.utils.nth_weekday_of_month` imported but unused

Remove unused import: `holidays.utils.nth_weekday_of_month`

(F401)